### PR TITLE
DCOS-39961: left align column names with bar

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
## Testing

Check if the table headers look the same as the mockup in the design linked in the jira

## Trade-offs

None that I'm aware of

## Dependencies

- [x] @lilysamimi or @leemunroe checking of this gorgeous screenshot (feel free to tick the box 👍) 
bildschirmfoto 2018-08-08 um 16 59 37

### CODE REVIEW

**Comment by leemunroe** [comment]
@DanielMSchmidt Feel free to tell me these are covered in separate PRs but some things I notice looking at the screenshot...

* Spacing between progress bar and number looks too much - it's hard to tie the number to the bar. I would expect 8px or max 16px — @lilysamimi did you have a recommendation?
* "DISK" should be "Disk" i.e. not all caps

**Comment by lilysamimi** [comment]
Following up on @leemunroe's comment:

- I recommend an 8px spacing between the progress bar and number. 
- I believe this is being fixed in the capitalization (I'll check now) but it DISK and MEM should both be "Disk" and "Mem"

File: plugins/nodes/src/js/components/NodesGridView.js [hunk 1]
```diff
@@ -86,8 +86,7 @@ var NodesGridView = React.createClass({
 
     // Add 'Others' node to the list
     if (activeServiceIds.length > MAX_SERVICES_TO_SHOW) {
-      var classNameOther =
-        `dot service-color-${OTHER_SERVICES_COLOR}`;
+      var classNameOther = `dot service-color-${OTHER_SERVICES_COLOR}`;
```
Comment (juliangieseke) [inline plugins/nodes/src/js/components/NodesGridView.js]: prettier? :D

**Comment by juliangieseke** [comment]
Capitalization is covered in https://jira.mesosphere.com/browse/DCOS-40385 👍

File: plugins/nodes/src/js/components/NodesGridView.js (hunk 1 above)
Comment (lilysamimi) [reply plugins/nodes/src/js/components/NodesGridView.js]: I can't really see a difference in the UI. 🤔 https://cl.ly/3A1A2Y0m093u

**Comment by GeorgiSTodorov** [comment]
@juliangieseke Can't we use a right-aligned cell to reduce the distance between the progress bar and the percent.

### Before
screenshot from 2018-08-13 11-24-21

### After
screenshot from 2018-08-13 11-21-30

**Comment by juliangieseke** [comment]
now headline & bar arent aligned anymore - aligning them again is tedious and I dont think the approach is any better than merging the columns 🤔

**Comment by GeorgiSTodorov** [comment]
I tested a right-aligned header, but the problem is that it moves when hovered because of the caret.

Should we create a new jira for merging the cells?

**Comment by DanielMSchmidt** [comment]
I'll merge the columns and we'll see how to go on from there 👍 This issue is back in Progress, so let's please not waste any more precious review time on it right now :)

**Comment by DanielMSchmidt** [comment]
bildschirmfoto 2018-08-13 um 11 37 59

I found in the table styles that we use 6px for content spacing, so I used that, I hope this is okay. I understand that the percentage should be right aligned, but that is not really feasible right now. If we really want this I would like to see it as a separate Ui-kit topic to implement such a functionality.

**Comment by leemunroe** [comment]
Sounds good. Thanks for tracking @juliangieseke

**Pull request merged**